crates/testing/src/tests.rs

```diff
@@ -1,4 +1,77 @@
-#[test]
-const fn dummy() {
-    // kept here to prevent the ci from failing on no tests
+use std::{collections::BTreeMap, sync::LazyLock};
+
+use alloy::{
+    eips::eip7702::Authorization,
+    primitives::{b256, Address, B256, U256},
+    providers::{PendingTransactionBuilder, Provider, ProviderBuilder},
+    signers::SignerSync,
+};
+use alloy_network::{TransactionBuilder, TransactionBuilder7702};
+use alloy_signer_local::PrivateKeySigner;
+use reth::rpc::types::TransactionRequest;
+use url::Url;
+
+static REPLICA_RPC: LazyLock<Url> = LazyLock::new(|| {
+    std::env::var("REPLICA_RPC")
```

Comment:
i'm wondering if we could just skip the tests instead of failing them when u run these locally, not particularly great ux running `cargo test` and having them fail :( same issue in foundry. not a blocker though